projects/packages/forms/tests/php/contact-form/Contact_Form_Block_Test.php

```diff
@@ -57,15 +61,175 @@ public static function data_provider_test_register_child_blocks() {
 		return array(
 			'jetpack/input'   => array(
 				'jetpack/input',
+				array(
+					'__experimentalBorder' => array(
+						'color'  => true,
+						'radius' => true,
+						'style'  => true,
+						'width'  => true,
+					),
+					'color'                => array(
+						'text'       => true,
+						'background' => true,
+						'gradient'   => true,
+					),
+					'typography'           => array(
+						'fontSize'                     => true,
+						'lineHeight'                   => true,
+						'__experimentalFontFamily'     => true,
+						'__experimentalFontWeight'     => true,
+						'__experimentalFontStyle'      => true,
+						'__experimentalTextTransform'  => true,
+						'__experimentalTextDecoration' => true,
+						'__experimentalLetterSpacing'  => true,
+					),
+				),
 			),
 			'jetpack/label'   => array(
 				'jetpack/label',
+				array(
+					'color'      => array(
+						'text'       => true,
+						'background' => true,
+						'gradient'   => false,
+					),
+					'typography' => array(
+						'fontSize'                     => true,
+						'lineHeight'                   => true,
+						'__experimentalFontFamily'     => true,
+						'__experimentalFontWeight'     => true,
+						'__experimentalFontStyle'      => true,
+						'__experimentalTextTransform'  => true,
+						'__experimentalTextDecoration' => true,
+						'__experimentalLetterSpacing'  => true,
+					),
+				),
 			),
 			'jetpack/options' => array(
 				'jetpack/options',
+				array(
+					'spacing' => array(
+						'blockGap' => true,
```

Comment:
```suggestion
						'blockGap' => false,
```

I think this should be `false` as we're not adopting this block support prior to the release of the inner blocks refactor. It was explicitly opted out of via https://github.com/Automattic/jetpack/pull/42890/commits/95e38e7d070686a972fe588fc5b04171d0d33fe3 earlier this morning.

Reply:
Ah great. Slow news day for me. Thanks for the tip off.

Reply:
Well, I'd say half the issues are past oversights in keeping the JS and PHP registrations in sync. So most of isn't this PR's problem so to speak.